test/integrations/destinations/sprig/network.ts

```diff
@@ -0,0 +1,61 @@
+const deleteNwData = [
```

Comment:
Is there any case of 429 due which user deletion can fail.
Please make sure we have test cases for all the error codes